src/faq.njk

```diff
@@ -165,12 +166,31 @@ eleventyNavigation:
 
   <main>
     <dl class="faq__faq">
-      <dt>Why won't it start in VirtualBox?</dt>
+      
+      <dt>I downloaded the developer preview liveCD. What is the username and password?</dt>
+      <dd class="faq__markdown">
+        <p>The login credentials for the liveboot currently is <em>liveuser</em>, all in small letters, and <b>no password</b>. Just type <em>liveuser</em> in the username field and press enter/click on the login button.</p>
```

Comment:
should be "currently are", I think? Credentials is plural.

Reply:
I think I'll go with keeping it singular(credentials -> credential, and are -> is ) since that makes more sense.